app/src/main/kotlin/com/wire/android/ui/e2eiEnrollment/GetE2EICertificateViewModel.kt

```diff
@@ -68,8 +68,7 @@ class GetE2EICertificateViewModel @Inject constructor(
                     .fold({
                         enrollmentResultFlow.emit(Either.Left(it))
                     }, {
-                        if (it is E2EIEnrollmentResult.Initialized) requestOAuthFlow.emit(it)
-                        else enrollmentResultFlow.emit(Either.Right(it))
```

Comment:
question: how is this change related to the fix?

Reply:
Actually it is not. But `it` could be only `E2EIEnrollmentResult.Initialized` here, so this `if` is not needed and AS annoys me by yellow highlighting ;P